about.html

```diff
@@ -0,0 +1,168 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="http://fonts.googleapis.com/css?family=Lato:100,300,400,700,900" type="text/css" >
+    <script src="https://kit.fontawesome.com/8d0a2aa6d4.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="css/about.css">
+    <title>About section</title>
+</head>
+<body>
+  <header class="header">
+    <ul id="header-list">
+        <li class="header-item icon"><a href="#"><i class="fa fa-facebook"></i></a></li>
+        <li class="header-item icon"><a href="#"><i class="fa fa-twitter" aria-hidden="true"></i></a></li>
+        <li class="header-item"><a href="#">English</a></li>
+        <li class="header-item"><a href="#">My Page</a></li>
+        <li class="header-item"><a href="#">Logout</a></li>
+    </ul>
+</header>
+<div class="head-container">
+  <div class="head-image">
+      <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+      </div>
+  <nav class="nav-container">
+      <div class="nav-logo">
+      <img src="images/my-logo.png" class="nav-logo-img" alt="home logo">
+      <a href="index.html" class="nav-head">Global Youth <br> Empowerment</a>
+      </div>
+      
+      <div class="nav-dir">
+          <ul class="nav-menu hide">
+            <li class="nav-item">
+              <a class="nav_link home-nav" href="index.html">Home</a>
+            </li>
+              <li class="nav-item">
```

Comment:
- [ ] [OPTIONAL] It  is redundant to have an about page link on your about page. I kindly suggest that you make this link an home page link instead.